demo/src/app/components/dropdown/demos/basic/dropdown-basic.html

```diff
@@ -2,7 +2,7 @@
   <div class="col">
     <div ngbDropdown class="d-inline-block">
       <button class="btn btn-outline-primary" id="dropdownBasic1" ngbDropdownToggle>Toggle dropdown</button>
-      <div class="dropdown-menu" aria-labelledby="dropdownBasic1">
+      <div ngbDropdownMenu aria-labelledby="dropdownBasic1">
```

Comment:
Second example here also requires `ngbDropdownMenu`!

Reply:
ngbDropdownMenu doesn't work to me. It had these issue. I didn't click the button but It showed me all the options.
![capture](https://user-images.githubusercontent.com/18093681/30288362-146db29e-96ee-11e7-90bd-f524cb44ea07.JPG)

Reply:
@GRTO please open a new issue following our submission guidelines (https://github.com/ng-bootstrap/ng-bootstrap/blob/master/README.md#you-think-youve-found-a-bug) if you believe that there is a bug to be fixed. Commenting on closed issues / PRs won't trigger any actions.